backend/src/crash/game_handler.py

```diff
@@ -207,6 +204,24 @@ def get_crash_event(self):
     def wait_for_crash(self):
         self.crash_event.wait()
 
+    def update_players_history(self):
+        for ws in self.players:
+            current_player = self.players[ws]
+            current_lobby_player = self.game_handler_parent.players[ws]
```

Comment:
Having the distinction of the two types of players might prove flimsy down the road, but works for now.

Reply:
Oh and same comment about the variable name (`current_lobby_player` -> `lobby_player`)